tests/pipeline/test_course_daily_metrics.py

```diff
@@ -111,40 +119,35 @@ def setup(self, db):
             ) for i, days in enumerate(self.cert_days_to_complete)]
 
     def test_get_enrolled_in_exclude_admins(self):
-        """
-
-        """
 
-        # Get the number of course enrollments
-        with mock.patch.dict('figures.helpers.settings.FEATURES', {'FIGURES_IS_MULTISITE': False}):
```

Comment:
In this test module/class, we're getting rid of single vs multisite awareness in tests as much as we can and doing the multisite support in the constructor if organizations supports multisite. _Note: the clear exception is when we have to tests explicit multisite handling code, like `figures.sites`.

We can make this work because Tox will test the different environments (meaning the community vs Appsembler fork of organizations (needed to support Tahoe))